server/routes/publicLists.js

```diff
@@ -0,0 +1,51 @@
+const express = require("express");
+const router = express.Router();
+const UserListModel = require("../models/userListModel");
+const db = require("../models");
+
+const getLists = async (req, res) => {
+  const userId = req.params.userId;
+  try {
+    const results = await UserListModel.findAll({
+      attributes: ["id", "title", "items", "imageUrl", "isPrivate", "userId"],
+      where: { userId: userId, isPrivate: false },
+      order: ["id"],
+    });
+    res.status(200).send(results);
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+const getProductsOfList = async (req, res) => {
+  const userId = req.params.userId;
+  const listId = req.params.listId;
+  try {
+    const listFound = await UserListModel.findOne({
+      where: {
+        id: listId,
+        isPrivate: false,
+        userId: userId,
+      },
+    });
+
+    if (listFound) {
+      const [products] = await db.query(`
+        SELECT "Products".name, "Products".link, "Products".id, "Products"."currentPrice", "Products"."previousPrice", "Products"."imageUrl" FROM "ListProducts" 
+        JOIN "Products" ON ("ListProducts"."productId" = "Products".id)
+        JOIN "UserLists" ON ("ListProducts"."listId" = "UserLists".id)
```

Comment:
Is this line necessary? If you were basing it off the one I wrote in the lists route, it was necessary there because I was having to check to make sure that the associated `UserList` belonged to the user, which isn't a factor here.